bot/exts/info/pypi.py

```diff
@@ -67,8 +67,14 @@ async def get_package_info(self, ctx: Context, package: str) -> None:
                     log.trace(f"Error when fetching PyPi package: {response.status}.")
 
         if error:
-            await ctx.send(embed=embed, delete_after=INVALID_INPUT_DELETE_DELAY)
-            await ctx.message.delete(delay=INVALID_INPUT_DELETE_DELAY)
+            error_message = await ctx.send(embed=embed)
+            await wait_for_deletion(error_message, (ctx.author.id,), timeout=INVALID_INPUT_DELETE_DELAY)
+
+            # If won't ghost-ping when deleting message 
```

Comment:
Please look over this again haha

Reply:
Was referring to the fact I didn't need to import `bot.constants.Emojis` anymore. Import was removed